tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala

```diff
@@ -328,6 +338,7 @@ trait WebActionsApiTests extends ControllerTestCommon with BeforeAndAfterEach wi
 
         it should s"not match invalid routes (auth? ${creds.isDefined})" in {
             implicit val tid = transid()
+            customOptions = true
```

Comment:
i dont follow - to clarify "do we need more than two" i was thinking that we only need two tests for options: one without the default annotation and one with.